CONTRIBUTING.md

```diff
@@ -462,11 +508,3 @@ Follow these steps to add the `.html` link to the navigation bar upon rendering.
 6) Replace  `tech-section`, `analysis_file_name` with the corresponding file names.  
 7) Save the file!  
 8) After you [render the notebook with snakemake](#rendering-notebooks), test the link to make sure it works.  
-
-## Pull request status checks
```

Comment:
This seemed kind of out of date now and better covered by the more specific github actions descriptions